src/endpoints/routing.cc

```diff
@@ -544,6 +544,14 @@ api::plan_response routing::operator()(boost::urls::url_view const& url) const {
     utl::verify(tt_ != nullptr && tags_ != nullptr,
                 "mode=TRANSIT requires timetable to be loaded");
 
+    auto const kMaxResults = config_.timetable_
```

Comment:
```suggestion
    auto const kMaxResults = config_.timetable_
```

`k*` variables are reserved for compile-time constants